lib/loader.js

```diff
@@ -220,7 +220,6 @@ class Component {
     // "Expose" operations by omitting the leading _ from the method name.
     //
     Object.keys(endpoint.pathItem)
-      .filter(key => endpoint.pathItem[key].operationId)
```

Comment:
So I'm not really sure why this filter existed in the first place but it seems fine to remove?